test/k8s-e2e/convtojson.py

```diff
@@ -54,7 +54,7 @@ def convert_junit_01_xml_to_json(
                 res['name'] = res['name'].strip()
                 matchObj = re.match(r'\[(.*?)\].*', res['name'], re.M | re.I)
                 res['sig'] = matchObj.group(1).strip()
-                res['duration'] = float(res['time'])
+                res['duration'] = int(float(res['time']))
```

Comment:
Why do you need a number not just leaving the string?

Reply:
removing number after the decimal point makes the time values more readable.